babel.config.js

```diff
@@ -3,5 +3,6 @@ module.exports = function( api ) {
 
 	return {
 		presets: [ '@wordpress/babel-preset-default' ],
+		plugins: [ 'babel-plugin-inline-json-import' ],
```

Comment:
Curious: Why do we do this in Babel, and not via Webpack? In fact, the [documentation for `json-loader`](https://webpack.js.org/loaders/json-loader/) claims this is enabled by default. Seems like we shouldn't need anything extra to just `import foo from './foo.json';` ? Or does it not work for Parcel?

Generally I think of Babel plugins to not work much with the filesystem, and defer that role to the bundler.

Reply:
Good question. To explain it, I need to give the full picture of how things work as of today. We have to steps in the build:
1. we use Babel to generate `build` and `build-module` folders for package
2. we use webpack to reuse `build-module` to generate build files consumed by Gutenberg

If we would defer to webpack, we would be publishing to npm code that couldn't be used in any project without having setup for loading JSON files. This is one of the reasons why we stopped importing CSS files inside JS files. That's why I assumed it would be important to ensure that it is still possible to consume `build` folder of `block-library` package without enforcing projects to handle JSON files.

In addition, I anticipate that to support internationalization (i18n) we will have to rewrite the content of inlined JSON file and this is where Babel shines.

Reply:
> Good question. To explain it, I need to give the full picture of how things work as of today. We have to steps in the build:

That all makes sense. Thanks for explaining.